Makefile

```
@@ -74,46 +132,52 @@ webpack:
 	npm run dev
 
 elevate:
-	$(wagtail) python manage.py elevate_sso_user_permissions --email=$(email)
+	$(wagtail-exec) python manage.py elevate_sso_user_permissions --email=$(email)
 
 collectstatic:
-	$(wagtail) python manage.py collectstatic
+	$(wagtail-exec) python manage.py collectstatic
 
 findstatic:
-	$(wagtail) python manage.py findstatic $(app)
+	$(wagtail-exec) python manage.py findstatic $(app)
 
 bash:
-	$(wagtail) bash
+	$(wagtail-exec) bash
 
 requirements:
-	$(wagtail) poetry export --without-hashes --output requirements.txt
+	$(wagtail-exec) poetry export --without-hashes --output requirements.txt
 
 superuser:
-	$(wagtail) python manage.py shell --command="from django.contrib.auth import get_user_model; get_user_model().objects.create_superuser('admin', email='admin', password='password', first_name='admin', last_name='test')"
+	$(wagtail-exec) python manage.py shell --command="from django.contrib.auth import get_user_model; get_user_model().objects.create_superuser('admin', email='admin', password='password', first_name='admin', last_name='test')"
 
 fixtree:
-	$(wagtail) python manage.py fixtree
+	$(wagtail-exec) python manage.py fixtree
 
 menus:
-	$(wagtail) python manage.py create_menus
+	$(wagtail-exec) python manage.py create_menus
 
 index:
-	$(wagtail) python manage.py update_index
+	$(wagtail-exec) python manage.py update_index
 
 listlinks:
-	$(wagtail) python manage.py list_links
+	$(wagtail-exec) python manage.py list_links
 
 wagtail-groups:
-	$(wagtail) python manage.py create_groups
+	$(wagtail-exec) python manage.py create_groups
 
 pf-groups:
-	$(wagtail) python manage.py create_people_finder_groups
+	$(wagtail-exec) python manage.py create_people_finder_groups
 
 create_section_homepages:
-	$(wagtail) python manage.py create_section_homepages
+	$(wagtail-exec) python manage.py create_section_homepages
+
+reset-db:
+	docker-compose stop db
```

```suggestion
	docker-compose kill db
```
not much in it but this might be slightly quicker?